src/main/scala/cromwell/util/TryUtil.scala

```diff
@@ -7,6 +7,12 @@ import com.typesafe.scalalogging.LazyLogging
 import scala.concurrent.duration.Duration
 import scala.util.{Success, Failure, Try}
 
+case class AggregatedException[A](exceptions: Seq[Failure[A]]) extends Exception {
+  override def getMessage: String = {
+    exceptions.map(_.exception.getMessage).mkString("\n")
+  }
+}
```

Comment:
Note that validations don't need to have a string on the left, it can be anything, including a throwable. The only real difference is that it's an applicative instead of a monad which allows you to do an accumulation vs. a fail-fast.